bootstrap/src/main/java/se/fortnox/reactivewizard/SetupModule.java

```diff
@@ -0,0 +1,33 @@
+package se.fortnox.reactivewizard;
+
+import com.google.inject.Binder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Hooks;
+import se.fortnox.reactivewizard.binding.AutoBindModule;
+
+/**
+ * Module for setting up the environment.
+ */
+public class SetupModule implements AutoBindModule {
+    private static final Logger LOG = LoggerFactory.getLogger("Hooks.onErrorDropped");
+
+    @Override
+    public void configure(Binder binder) {
+        Hooks.onErrorDropped(throwable -> {
+            logError(LOG, throwable);
+        });
+    }
+
+    /**
+     * package private to be able to be tested without too much trouble.
+     *
+     * @param logger the logger to be used
+     * @param throwable the exception
+     */
+    void logError(Logger logger, Throwable throwable) {
+        logger.warn("Tried to send item or error to subscriber but the subscriber had already left. " +
```

Comment:
unsure if this would be cleaner: 


The error has been dropped because it cannot be propagated through the reactive chain.  This happens when a publisher calls onError on the operator despite having already called onCompleted on it previously. This may happen when an error is generated by different places in the reactive chain and only one of them can be propagated and resulting in this exception being dropped.

Reply:
I'll merge our two messages